generators/server/templates/src/main/java/package/service/UserService.java.ejs

```diff
@@ -1042,8 +1044,10 @@ public class UserService {
         }
         if (details.get("email") != null) {
             user.setEmail(((String) details.get("email")).toLowerCase());
+        } else if (sub.contains("|")) { // special handling for Auth0
```

Comment:
Can we also check if username contains `@` ?

Reply:
Added.